metrics/tarantool/memtx.lua

```diff
@@ -120,6 +120,36 @@ local function update()
             'Amount of bytes used by retained `read_view` tuples',
             memtx_stat.mvcc.tuples.read_view.retained.total, {kind = "total"}, nil, {default = true})
 
+    -- Tarantool 3.0 memory statistics
+
+    local ok, memtx_stat_3 = pcall(box.stat.memtx)
+    if not ok or memtx_stat_3.data == nil or memtx_stat_3.index == nil then
+        return
+    end
+
+    collectors_list.memtx_tuples_data_total =
+        utils.set_gauge('memtx_tuples_data_total',
+            'Total amount of memory allocated for data tuples',
+            memtx_stat_3.data.total, nil, nil, {default = true})
+    collectors_list.memtx_tuples_data_read_view =
+        utils.set_gauge('memtx_tuples_data_read_view',
+            'Memory held for read views',
+            memtx_stat_3.data.read_view, {kind = "read_view"}, nil, {default = true})
+    collectors_list.memtx_tuples_data_garbage =
+        utils.set_gauge('memtx_tuples_data_garbage',
+            'Memory that is unused and scheduled to be freed',
+            memtx_stat_3.data.garbage, nil, nil, {default = true})
+
+
+    collectors_list.memtx_index_extents_total =
```

Comment:
What's the idea behind "extents" part of the name?

Reply:
I took descriptions from this issue:
https://github.com/tarantool/doc/issues/3422

Reply:
I think just `memtx_index_total` is more descriptive and easy to understand.